__tests__/mocks/FakeResponseHandler.ts

```diff
@@ -0,0 +1,9 @@
+export class FakeResponseHandler {
+  status!: () => this;
+  json!: (response: any) => any;
```

Comment:
why use "!" if the typescript automatically detects that a variable cannot be null or undefined if it has a defined type?

Reply:
I didn't understand this class, could you explain it to me?
Another thing, wouldn't it be better to use MockResponseHandler when using FakeResponseHandler?

Reply:
This class is responsible to make possible create unity tests for ```ResponseHandler``` since it depends on the ```Response``` object provided by the ```express.Router```. By this way we can mock it to use the ```send``` method from ```ResponseHandler```. I agree with your naming suggestion

Reply:
Understood, thank you